packages/js/src/feeds/Feeds.ts

```diff
@@ -1,9 +1,34 @@
-import { NovuEventEmitter } from '../event-emitter';
+import type { PaginatedResponse } from '../types';
+import { BaseModule } from '../utils/base-module';
+import { Notification } from './notification';
 
-export class Feeds {
-  #emitter: NovuEventEmitter;
+interface FetchFeedOptions {
+  page?: number;
+  feedIdentifier?: string | string[];
+  seen?: boolean;
+  read?: boolean;
+  limit?: number;
+  payload?: Record<string, unknown>;
+}
+
+export class Feeds extends BaseModule {
+  async fetch({ page = 1, ...restOptions }: FetchFeedOptions): Promise<PaginatedResponse<Notification>> {
+    return this.callWithSession(async () => {
+      try {
+        this._emitter.emit('feeds.fetch.pending');
+
+        const response = await this._apiService.getNotificationsList(page, restOptions);
+        const modifiedResponse: PaginatedResponse<Notification> = {
+          ...response,
+          data: response.data.map((el) => new Notification(el)),
```

Comment:
wrap the data with Notification class that we will use later for the ORM pattern